docs/port.md

```diff
@@ -4,18 +4,23 @@ To the ports in the network namespace to the host network namespace, `--port-dri
 
 The default value is `none` (do not expose ports).
 
-| `--port-driver`      |  Throughput | Source IP
-|----------------------|-------------|----------
-| `slirp4netns`        | 6.89 Gbps   | Propagated
-| `socat` (Deprecated) | 7.80 Gbps   | Always 127.0.0.1
-| `builtin`            | 30.0 Gbps   | Always 127.0.0.1
+| `--port-driver`      |  Throughput | Source IP | Notes
+|----------------------|-------------|----------|-------
+| `slirp4netns`        | 2.51 Gbps   | Propagated | 
```

Comment:
How did you get this data?
https://github.com/rootless-containers/rootlesskit/actions/runs/17831573423/job/50738323576?pr=522 seems achieving 9.65 Gbps and 9.78 Gbps (detach-netns mode)